### ISSUE TITLE
error.vue $computed not found and no includes property on computedRef

### ISSUE THREAD

**Issue by isimmons** [nuxt/movies#19]
Hi,

I copied the error.vue page into my own project from the Mastering Nuxt 3 course. It does not work for me. 

I added lang="ts" to the script tag and ts errors tells me 2 things wrong.

`Cannot find name $computed. Did you mean computed?`

and

`Property 'includes' does not exist on type computedRef`  referring to message?.includes

If I remove those it works.

```
const message = computed(() => String(props.error?.message || ''))
const is404 = computed(() => props.error?.statusCode === 404)
```

**Comment by danielroe** [comment]
In future you can open a discussion on https://github.com/nuxt/framework.

I think probably you should enable the reactivity transform in your `nuxt.config`:

```js
export default defineNuxtConfig({
  experimental: {
    reactivityTransform: true
  }
})
```

**Comment by isimmons** [comment]
Thank you @danielroe 

Adding that to my nuxt.config gave me a 500 server error "Nuxt not available"  I tried restarting the dev server.

I noticed this repo is on one of the RC 13 commits. Since RC13 officially released a couple of hours ago, I upgraded and the 500 error went away but then [this tailwind issue](https://github.com/nuxt-modules/tailwindcss/issues/480) came up and I had to manually update @nuxt/tailwindcss to the latest version. But all is good now. Thanks again
